.github/CONTRIBUTING.md

```diff
@@ -96,7 +96,19 @@ If you are adding new assets that are not already explicitly licensed under CC-B
 
 If at any point you are confused or unsure of an asset's license or our policy, ask a Maintainer to help you.
 
-## 7. Banned Content
+## 7. Pull Request Standards
+
+You should complete the pull-request template in its entirity.
+
+Any pull-request that does not adequately complete the pull-request template will not be merged and may be closed by maintainers (It is up to the discretion of the maintainers to decide if the PR will be accepted, closed or marked as 'do not merge').
+ - Balance changes must indicate that they affect the balance of the game.
+ - Changes must be documented in their entirity, and the full extend of their effects mentioned. (For example, if you change it so all mobs are half speed, don't label the PR as 'monkeys now move twice as slow'). Failing to document the full extent of the changes may be seen as malicious and could result in a repo-ban in a worst case scenario.
+ - The section labeled 'about this pull request' should state the changes that the pull-request makes.
+ - The section labeled "why it's good for the game" should include the reasons behind the changes and how they will be good for the game.
+ - The testing section should contain screenshots, videos or a description showing that the PR works as specified. Pull-requests that ignore this section or are not tested may be closed by maintainers. This applies to small PRs that may seem trivial.
```

Comment:
Now ok we all know this section has been ignored almost entirely since it got introduced(mainly cause its a hassle after hours of work writing down an essay of all possible stuff you did to test it) trough as long as a completly text documentation is (in most cases at least) sufficient i guess its fine.

Reply:
For it to be respected template needs to be improved.
It's currently not really readable and ignored by most.

Reply:
You just have to put a screenshot in it showing that you have opened it and done at least a basic test on it. An essay isn't required

Reply:
> (mainly cause its a hassle after hours of work writing down an essay of all possible stuff you did to test it)

A basic set of reproduceable testing procedures is really all you need. Adding a screenshot or a video demonstrating that the thing works is equally acceptable and requires fairly minimal effort.

Reply:
In that case yeah sounds good